markets/bfp-market/contracts/storage/Position.sol

```diff
@@ -295,7 +302,13 @@ library Position {
             );
 
             // Check the minimum credit requirements are still met.
-            validateMinimumCredit(market, params.oraclePrice, marketConfig, addresses);
+            validateMinimumCredit(
```

Comment:
You should also test that the same revert occurs on settlement if min credit checks had passed at the time of commitment.